.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,17 @@
+name: Github Actions
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
```

Comment:
You can use container to make this more cleaner. You can check for example: https://github.com/gengjiawen/node/blob/88a7cf74b6702b82ba485e4d3393d162951d2a37/.github/workflows/CI.yml#L27.

Reply:
Yeah, i know it. I did it using the default template from github actions, the idea was make it like a fresh start.